jck/runtime.api/playlist.xml

```diff
@@ -392,6 +392,11 @@
 			<plat>x86-64_mac</plat>
 			<version>16</version>
```

Comment:
does something like `16+` cover `17` and beyond?

Reply:
@renfeiw would be the best person to answer that.

Reply:
Yes